### PR TITLE
Props

### PR DESCRIPTION
**Pull request by Apxucmpax**
(Props it is a mean of transmitting of the data from parent element to child element)

   6.1 State is situated inside class component.

   6.2 Props can not be changed.

   6.3 State can be changed.

   6.4 key input in cycle.

### CODE REVIEW

**Comment by Hamed-Fouladi** [comment]
I'm agree/ you can merge

**Comment by MrDSparky** [comment]
Im agree, you can merge this

File: src/App.js [hunk 1]
```diff
@@ -8,11 +8,33 @@ import {
 // const Obj = module.require('../src/components/lifeTimeHooks');
 // const Parent = Obj.Parent;
 // const Child = Obj.Child;
+// array data
+let data = [
```
Comment (SergyiKrasnytskyi) [inline src/App.js]: Const  variable needed

**Review by MrDSparky — APPROVED** [review]

**Review by SergyiKrasnytskyi — APPROVED** [review]

**Review by HelennnSol — APPROVED** [review]

**Review by Hamed-Fouladi — APPROVED** [review]
Everything is fine

**Comment by ZachepaRostislav** [comment]
add to clause 6.2 that the props is read-only

**Pull request merged**